twenty-first/src/mock/mmr/mock_mmr.rs

```diff
@@ -1,33 +1,34 @@
 use std::marker::PhantomData;
 
-use leveldb::batch::WriteBatch;
-
 use crate::shared_math::digest::Digest;
-use crate::storage::storage_vec::traits::*;
-use crate::storage::storage_vec::RustyLevelDbVec;
 use crate::util_types::algebraic_hasher::AlgebraicHasher;
 use crate::util_types::shared::bag_peaks;
 use crate::utils::has_unique_elements;
 
-use super::mmr_accumulator::MmrAccumulator;
-use super::mmr_membership_proof::MmrMembershipProof;
-use super::mmr_trait::Mmr;
-use super::shared_advanced;
-use super::shared_basic;
+use crate::util_types::mmr::{
+    mmr_accumulator::MmrAccumulator, mmr_membership_proof::MmrMembershipProof, mmr_trait::Mmr,
+    shared_advanced, shared_basic,
+};
 
+/// MockMmr is available for feature `mock` and for unit tests.
+///
+/// It implements an in-memory Mmr using a Vec<Digest> as the data store.
```

Comment:
You should clarify that this is an archival-MMR, not an MMR accumulator. An archival MMR stores *all* nodes in multiple trees whereas an MMR accumulator only stores the roots of the binary trees.

This clarification could take place either in the documentation, or through the name of the mock struct. Up to you.